### ISSUE TITLE
Add a ploy.conf plain sample after explain the options

### ISSUE THREAD

**Issue by oscarmlage** [ployground/bsdploy#48]
after plain options, I think a basic sample, how a simple ploy.conf would looks could be great to correctly understand the meaning.

**Comment by tomster** [comment]
i agree, in fact i think i need to create a dedicated page where all configuration options are listed and explained. so far i just mention them by and by as i touch on them in the narrative documentation...
